user/build-matrix.md

```diff
@@ -216,6 +216,43 @@ script: env $EXTRA_TESTS ./test.py $TEST_SUITE
 ```
 {: data-file=".travis.yml"}
 
+### Explicitly included jobs with only one element in the build matrix
+
+As a special case, if your build matrix has only one element _and_ you specified
```

Comment:
Committed a similar simpler wording.

The documentation seems to use "explicitly included jobs" as the official term for the `matrix: include:` construct. So I try to stick to that constant phrase rather than inflect it -- to make the meaning clear without the need for long explanations each time.